controllers/drplacementcontrol.go

```diff
@@ -1865,6 +1865,11 @@ func (d *DRPCInstance) ensureVRGManifestWorkOnClusterDeleted(clusterName string)
 	}
 
 	d.log.Info("Request not complete yet", "cluster", clusterName)
+
+	if d.instance.Spec.ProtectedNamespaces != nil && len(*d.instance.Spec.ProtectedNamespaces) > 0 {
```

Comment:
We need to handle something similar in `cleanupForVolSync` which is Volsync specifc?

Reply:
I would like to handle that in #1391 and merge this for now as this fixes RBD.